tests/test_testclient.py

```diff
@@ -1,8 +1,10 @@
+from __future__ import annotations
+
 import itertools
 import sys
-from asyncio import current_task as asyncio_current_task
+from asyncio import Task, current_task as asyncio_current_task
 from contextlib import asynccontextmanager
-from typing import Callable
+from typing import Any, AsyncGenerator, Awaitable, Callable, Union
```

Comment:
Remove Any, please.

Use `|` instead of `Union`.

Reply:
In this file, Any is used only here:

`def current_task() -> Task[Any] | trio.lowlevel.Task:`

I couldn't figure it out what type should be there instead of Any. Could you help me on this one please?